fdm-app/app/components/import/changelog1.tsx

```
@@ -0,0 +1,108 @@
+                            <div className="flex flex-col">
+                                <h2 className="mb-3 text-lg leading-tight font-bold text-foreground/90 md:text-2xl">
+                                    {entry.title}
+                                </h2>
+                                <p className="text-sm text-muted-foreground md:text-base">
+                                    {entry.description}
+                                </p>
+                                {entry.items && entry.items.length > 0 && (
+                                    <ul className="mt-4 ml-4 space-y-1.5 text-sm text-muted-foreground md:text-base">
+                                        {entry.items.map((item, itemIndex) => (
+                                            <li
+                                                key={itemIndex}
```

Biome (2.1.2)

[error] Avoid using the index of an array as key property in an element.

This is the source of the key value.

The order of the items may change, and this also affects performances and component state.
Check the React documentation. 

(lint/suspicious/noArrayIndexKey)